src/main/java/seedu/address/model/transaction/Transaction.java

```diff
@@ -0,0 +1,105 @@
+package seedu.address.model.transaction;
+
+//import javax.management.Descriptor;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+/**
+ * Represents a Transaction in the sales book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+
+public class Transaction {
+
+    private final Description description;
+    private final Value value;
+    private final TxnStatus status;
+    private final Owner owner;
```

Comment:
Just a note: The owner of the transaction will eventually be dictated by `PersonTransactionMap` (to be implemented in v1.3), so there will be no need to store this in the Transaction class. 

However, indicating the owner (of Person class) when calling the AddTransaction command will still be required, and we can consider calling the index of the owner.